### PR TITLE
Johnc imputesex docsfixes

### PR DESCRIPTION
**Pull request by johnc1231**
Mostly just adding impute_sex documentation from the markdown, but also fixed two small typos in other locations.

### CODE REVIEW

File: python/hail/dataset.py [hunk 1]
```diff
@@ -2000,6 +2000,37 @@ def impute_sex(self, maf_threshold=0.0, include_par=False, female_threshold=0.2,
         """Impute sex of samples by calculating inbreeding coefficient on the
         X chromosome.
 
+        **Examples**
+
+        >>> imputed_sex_vds = vds.impute_sex()
```
Comment (jigold) [inline python/hail/dataset.py]: Can you use the following example instead:

Remove samples where imputed sex does not equal reported sex:
```
>>> imputed_sex_vds = (vds.impute_sex()
...       .annotate_samples_expr('sa.sexcheck = sa.pheno.isFemale == sa.imputesex.isFemale')
...       .filter_samples_expr('sa.sexcheck || isMissing(sa.sexcheck)'))

File: python/hail/dataset.py [hunk 2]
```diff
@@ -234,7 +234,7 @@ def aggregate_intervals(self, input, expr, output):
 
         - ``interval`` (*Interval*): genomic interval
         - ``global``: global annotations
-        - ``variants`` (*Aggregable[Variant]*): aggregable of :ref:`variant`s Aggregator namespace below.
+        - ``variants`` (*Aggregable[Variant]*): aggregable of :py:class:`~hail.representation.Variant` s Aggregator namespace below.
```
Comment (jigold) [inline python/hail/dataset.py]: Not for you to fix with this PR -- we should come to a consensus on whether we want to refer to the Python type or the expression language type (coming soon).

**Comment by johnc1231** [comment]
Fixed the example, back to you.

**Comment by jigold** [comment]
Can you add the example header I gave?

Remove samples where imputed sex does not equal reported sex:

**Comment by johnc1231** [comment]
Sorry for leaving that out. My bad. Should be good now

**Pull request merged**